std/process.d

```diff
@@ -80,8 +80,8 @@ License:
 Source:
     $(PHOBOSSRC std/_process.d)
 Macros:
-    OBJECTREF=$(D $(LINK2 object.html#$0,$0))
-    LREF=$(D $(LINK2 #.$0,$0))
+    OBJECTREF=`$(LINK2 object.html#$0,$0)`
```

Comment:
I'm also of the opinion that explicit links should not need to be wrapped in anything. If you find yourself always writing `a(b(c))`, then define `d(c) => a(b(c))` and use `d`... which is what it is actually doing here, but it is repeated work on module level from one of the global macro definition files. I guess that's a separate issue from this PR, but these local ref macros should all be replaced with one of the generic ref things, that can style the text in the monospace font using a specific class name rather than wrapping the two macros together.